akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FileUploadDirectivesSpec.scala

```diff
@@ -147,7 +148,7 @@ class FileUploadDirectivesSpec extends RoutingSpec {
   }
 
   private def read(file: File): String = {
-    val in = new FileInputStream(file)
+    val in = Files.newInputStream(file.toPath)
```

Comment:
I think this whole method works only accidentally (if at all). It's only in tests, so maybe no need to fix it right now.